src/Extension/FluentExtension.php

```diff
@@ -278,10 +263,18 @@ public function augmentSQL(SQLSelect $query, DataQuery $dataQuery = null)
             }
         }
 
-        // Get default provider
-        $providerName = Config::inst()->get(get_class($this->owner), 'defaultProvider');
-        /** @var DefaultProvider $defaultProvider */
-        $defaultProvider = $providerName ? Injector::inst()->get($providerName) : null;
+        // On frontend, ensure at least one localised version exists in localised base table (404 otherwise)
+        if (FluentState::singleton()->getIsFrontend()) {
+            $wheres = [];
+            $baseTableLocalised = $this->owner->baseTable() . '_' . self::SUFFIX;
```

Comment:
I think this might be useful to make into its own method, i.e. `$somethingFluent->getLocalisedTable()`. I can imagine user code taking advantage of this